commercial/app/controllers/HostedContentController.scala

```diff
@@ -49,6 +49,8 @@ class HostedContentController(
     } recover {
       case e: ContentApiError if e.httpStatus == 410 =>
         cached(commercialExpired(wasAHostedPage = true))
+      case e: ContentApiError if e.httpStatus == 404 =>
+        Cached(cacheDuration)(WithoutRevalidationResult(NotFound))
```

Comment:
We'll now cache these 404s for a minute. The full `Cache-Control` header now looks like this:

`Cache-Control: max-age=60, stale-while-revalidate=6, stale-if-error=864000`

Reply:
Although not explicit, `Cache-Control: public` is implied by the use of `max-age=60`, as per the documentation:

> The max-age directive on a response implies that the response is cacheable (i.e., "public") unless some other, more restrictive cache directive is also present.

https://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.9.3